backend/app/converters/ead_converter.rb

```diff
@@ -253,7 +253,7 @@ def self.configure
         end
         content = langmaterial.to_s
 
-        unless content.nil? || content == ''
+        unless content.nil? || content.gsub(/\s+/, '') == ''
```

Comment:
@lorawoodford I think this is functionally equivalent to `content.strip.empty?` which might be a bit clearer and catch things like newlines only. Worth an update?

Reply:
Done!